data_structures/src/Stack/Stack.cs

```diff
@@ -6,17 +6,18 @@ namespace ADT
 {
     public class Stack<T> : IStack<T>
     {
-        private T[] _items { get; set; }
-        private int _topOfStack { get; set; }
+        private readonly T[] _items;
```

Comment:
Use local variables for things that don't do any calculations or aren't a part of the class's public interface. Also, use immutable types whenever possible.